packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/corellium/api.py

```
@@ -0,0 +1,171 @@
+        """
+        Delete a virtual instance.
+
+        Does not return anything since Corellium's API return a HTTP 204 response.
+        """
+        try:
+            res = self.req.delete(f'{self.baseurl}/v1/instances/{instance.id}')
+            data = res.json() if res.status_code != 204 else None
+            res.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            msgerr = data if data is not None else str(e)
+            raise CorelliumApiException(msgerr)
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)